### PR TITLE
Pin `safety` package version to `>=1.0.0` and `<2.0.0`

### LINKED ISSUE #29: 'NoneType' object has no attribute 'telemetry'
The hook at v1.2.4 has been failing for me since 12:14 a.m. UTC today (June 29th, 2022) in one repo but not others. I believe the underlying API is [reported as up](https://status.pyup.io/), and I'm stumped as to how to debug this further, so any help would be greatly appreciated. This is the error I'm getting:

```
Unexpected Exception happened: 'NoneType' object has no attribute 'telemetry'
Traceback (most recent call last):
  File "/home/runner/.cache/pre-commit/reponl89_evy/py_env-python3.10.4/lib/python3.10/site-packages/safety/cli.py", line 101, in check
    announcements = safety.get_announcements(key=key, proxy=proxy_dictionary, telemetry=ctx.parent.telemetry)
AttributeError: 'NoneType' object has no attribute 'telemetry'
```

### PR DESCRIPTION
**Pull request by sshishov**
This is `hotfix` to support old `safety` package

Fixes #29

### CODE REVIEW

**Review by Kurt-von-Laven — CHANGES_REQUESTED** [review]
Greatly appreciate the fixes @sshishov!

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     ],
     packages=find_packages('.'),
     install_requires=[
-        'safety',
+        'safety<2.0.0',
```
[the remark this answers is not in the document]
Comment (sshishov) [reply setup.py:24]: Thanks @Kurt-von-Laven , going to add the minimum pinned version as `1.0.0`. Do not want to do a hard pin as the mentioned product can still have some small bug fixes and patch versions.

[the remark this answers is not in the document]
Comment (sshishov) [reply setup.py]: Updated, thanks

**Review by Kurt-von-Laven — APPROVED** [review]
That seems perfectly reasonable. Thank you for putting up these fixes so quickly! They seem solid to me and very low-risk to merge considering the hook is presently dead in the water. I think the safest thing would be to merge this one first, do a release, and then merge the other one and do another release.

**Comment by sshishov** [comment]
@Kurt-von-Laven This fix is much more safe than another. I will add the note in another MR regarding my concerns

**Comment by Lucas-C** [comment]
Thank you for the PR!

Sadly this wasn't passing the GitHub Actions checks.

It has been superseded by [release v1.3.0](https://github.com/Lucas-C/pre-commit-hooks-safety/releases/tag/v1.3.0)

Closing this now.

**Pull request closed without merging**